admin/securedrop_admin/__init__.py

```diff
@@ -59,9 +74,18 @@
 MAX_NAMESERVERS = 3
 LIST_SPLIT_RE = re.compile(r"\s*,\s*|\s+")
 
-I18N_CONF = "securedrop/i18n.json"
+I18N_CONF_PATH = "/usr/share/securedrop-admin/i18n.json"
 I18N_DEFAULT_LOCALES = {"en_US"}
 
+READONLY_CONFIG_PATH = "/usr/share/securedrop-admin"
+ANSIBLE_PATH = os.path.join(READONLY_CONFIG_PATH, "ansible-base")
+TRANSLATIONS_PATH = os.path.join(READONLY_CONFIG_PATH, "translations")
+if OS_TYPE == OSType.TAILS:
```

Comment:
I'd need to check but it's likely that a `~/.securedrop-admin` path (or worst case scenario, a `~/.config/securedrop-admin` path) would work for both, as we require users to set dotfiles directories as persistent in tails anyway.